x/deployment/types/types.go

```diff
@@ -54,9 +54,10 @@ const (
 
 // GroupSpec stores group specifications
 type GroupSpec struct {
-	Name         string          `json:"name"`
-	Requirements []sdk.Attribute `json:"requirements"`
-	Resources    []Resource      `json:"resources"`
+	Name             string          `json:"name"`
+	Requirements     []sdk.Attribute `json:"requirements"`
+	Resources        []Resource      `json:"resources"`
+	OrderBidDuration int64           `json:"bid-duration"`
```

Comment:
add validation to `ValidateBasic()` - it should not be less than... I don't know... `1`.